binding/SkiaSharp/SKTypeface.cs

```diff
@@ -227,23 +227,14 @@ public ushort[] GetGlyphs (ReadOnlySpan<int> codepoints) =>
 		public ushort[] GetGlyphs (string text) =>
 			GetGlyphs (text.AsSpan ());
 
-		public ushort[] GetGlyphs (ReadOnlySpan<char> text)
-		{
-			using var font = ToFont ();
-			return font.GetGlyphs (text);
-		}
+		public ushort[] GetGlyphs (ReadOnlySpan<char> text) =>
+			GetFont ().GetGlyphs (text);
```

Comment:
Seeing as the surrounding methods (including some overloads) use `GetFont` instead of `ToFont` and because `SKFont.GetGlyphs` is a pure method, I think it is safe to use `ToFont` in this situation. If you disagree though, I can move the change to a separate PR for later review.